recipe/bld.bat

```diff
@@ -0,0 +1,19 @@
+set LIBRARY_PREFIX=%LIBRARY_PREFIX:\=/%
+set MSYSTEM=MINGW%ARCH%
+set MSYS2_PATH_TYPE=inherit
+set CHERE_INVOKING=1
```

Comment:
I think so.  Without it, the make system detection script doesn't work right.  @mingwandroid knows more.  I just copy/paste from him.